shared/common-adapters/hoc-timers.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react'
+
+export type TimerFunc = (func: () => void, timing: number) => number
+export type ClearTimerFunc = (id?: ?number) => void
+export type TimerProps = {
+  setTimeout: TimerFunc,
+  clearTimeout: ClearTimerFunc,
+  setInterval: TimerFunc,
+  clearInterval: ClearTimerFunc,
+}
+
+function clearId (clearFunc: (id: number) => void, array: Array<number>, id?: ?number): void {
```

Comment:
I tried setting clearFunc to ClearTimerFunc but Flow was complaining that `clearInterval